### ISSUE TITLE
Translation to other languages

### ISSUE THREAD

**Issue by Corea** [rougier/from-python-to-numpy#36]
Is there any proper way to translate this book to other language?
I would like to do that in Korean whenever I have some free time :)

**Comment by rougier** [comment]
I'm not sure. The obvious way would be to translate each '.rst' file but then I don't know how to cope with further modification. If anyone has experience on this...

**Comment by himito** [comment]
I am interested in the Spanish translation :)

**Comment by rougier** [comment]
@himito Nice ! Thank you.

I need to find the right way to organize the repo.

**Comment by himito** [comment]
Hi all :)

I was looking for a translation platform and I found http://zanata.org. OpenStack uses it, maybe it would be interesting to take a look in more detail :)

**Comment by rougier** [comment]
I think this is only for software, no ?

**Comment by Gebon** [comment]
In my opinion, it's very difficult to support more than one language for books. Each translation had to be updated each time, when original book is changed. So, I suppose, that each translator should fork this repo and work on his translation separately, without merging it with this repo.

When changes occurs here, the translators should merge it and make appropriate changes to their translations.

Moreover, there possibly more than one translators group to one language, which don't agree with each others translations. In this repo @rougier can make some translations list with links.

PS. My English isn't good. I hope, that you understand my thoughts

**Comment by rougier** [comment]
Yes. The problem is that when the book is updated, there is no real way to know if you need to change something in you translation. I was hoping of finding some tracking tools between the original and the translation such that translators can know when change is needed.

**Comment by Gebon** [comment]
Wow! I think that this task is too difficult to be solved for now. The only one way to determine whether there need to update the translation (that I can imagine) is to manually look at the changes. In common case, languages is too different from each other to track their synchronization automatically. It will be great, if someone find any kind of such tool.

**Comment by rougier** [comment]
Maybe we should have "edition" (first, second, etc) using specific tag and translators can target this or that edition (most likley the latest).

**Comment by himito** [comment]
Nice solution !

**Comment by rougier** [comment]
Ok, so let's wait for some more (minor) corrections of the current draft and I will release a "First Edition".

**Comment by SergeStinckwich** [comment]
This is what we are doing for Pharo by Example book translations.

**Comment by hemangsk** [comment]
Hey everyone 😄 I'm interested to translate the book in [Hindi](https://en.wikipedia.org/wiki/Hindi) language. Looking forward to the first edition release!

**Comment by rougier** [comment]
I'm waiting for a few more corrections and I'll release the first edition.

**Comment by huang-xu** [comment]
I'll try to translate it into Simplified Chinese. Just started.

**Comment by rougier** [comment]
Many corrections have been made since publication and if every potential reviewers (just add a thumb up), I can make an official release for version 1.0. Translations would then be for this 1.0 release.

**Comment by himito** [comment]
Hi !,
It would be interesting to check [crowdin](https://github.com/marketplace/crowdin) for translations :)

**Comment by rougier** [comment]
Thanks for the link. Would that work for a book (with lot of text).
And is it free for an open-access book ?

**Comment by himito** [comment]
It's free for open source projects :smile: . I have tested it for the documentation of the Poppy project and it works pretty well :) https://crowdin.com/project/poppy-docs

**Comment by mukimei** [comment]
I'd like to help translate it into Japanese!

**Comment by rougier** [comment]
@mukimei Thanks for the offer. I'm currently writing another book (https://github.com/rougier/scientific-visualization-book) and this time I target PDF. I would like to make a similar nice PDF for the numpy book and maybe if people are still interested in translation, we can start a project for converting the book in PDF with several version, one for each language. Just add a thumb up to my post if you're still willing to help.

**Comment by dabane-ghassan** [comment]
Hello! is it too late for translations? i'd love to make an arabic version of it!

**Comment by rougier** [comment]
It's never too late and it would be great to have an arabic version. There have been a lot of suggestions on what would be the best tool/platform to wrtite translations but I think we did not converge on a solution. Do you know how to handle this?
